data/brands/shop/supermarket.json

```diff
@@ -946,6 +946,8 @@
       "locationSet": {"include": ["001"]},
       "tags": {
         "brand": "Choppies",
+        "brand:wikidata": "Q5104860",
```

Comment:
"Area served | Botswana, Zimbabwe, Zambia" - https://en.wikipedia.org/wiki/Choppies 

Restricting locationSet may be nice to keep it from appearing elsewhere.

See https://github.com/osmlab/name-suggestion-index/blob/5e55b07e746c35337f38cbb20fccff1bfe64ed47/CONTRIBUTING.md#locationset